build.sbt

```diff
@@ -73,8 +91,48 @@ lazy val codacyAnalysisModels = project
   .disablePlugins(com.lucidchart.sbt.scalafmt.ScalafmtCorePlugin)
   .enablePlugins(JavaAppPackaging)
 
+lazy val apiSwaggerFile: File =
+  codacyApiClient.base / "target" / "api" / "codacy-api" / Dependencies.codacyApiVersion / "swagger.yaml"
+
+lazy val downloadCodacyToolsSwaggerFile = Def.task[Unit] {
+  if (!Files.exists(apiSwaggerFile.toPath)) {
+    val result: String =
+      scala.io.Source
+        .fromURL(url(s"https://dl.bintray.com/codacy/Binaries/${Dependencies.codacyApiVersion}/swagger.yaml"))
```

Comment:
We just added the first version on https://dl.bintray.com/codacy/api/codacy-api/18.4.0/ we can depend on that now! :D